pkg/controller/istiocsr/deployments_test.go

```diff
@@ -776,6 +778,24 @@ func TestUpdateArgList(t *testing.T) {
 				"cluster-id": "cluster-123_dev.local",
 			},
 		},
+		{
+			name: "istioDataPlaneNamespaceSelector not provided should not include argument",
+			updateIstioCSR: func(istiocsr *v1alpha1.IstioCSR) {
+				// IstioDataPlaneNamespaceSelector is empty, so argument should not be present
+			},
```

Comment:
nit: Do we need to initialize with the default?
```suggestion
```

Reply:
If it's not required, lets remove it.

Reply:
Removed, thanks